scripts/prepare.sh

```diff
@@ -3,11 +3,31 @@
 cd "$(dirname "$0")" || exit 1
 
 # You only need to run this script once to setup a cluster
+NAMESPACE="${NAMESPACE:-mobile-console}"
```

Comment:
If a developer follows [these steps](https://github.com/aerogear/mobile-developer-console#local-openshift-cluster) in the README:

```sh
./scripts/minishift_start.sh
./scripts/prepare.sh
./scripts/development.sh
```
`prepare.sh` will set everything up in the `mobile-console` namespace, but `development.sh` switches to `myproject`. I think this script should also be updated with the same namespace or `oc project myproject` could be removed altogether because it will already be in the correct namespace.

Reply:
yes, good point. I will update it.